admittance_controller/test/test_admittance_controller.hpp

```diff
@@ -76,10 +76,14 @@ class TestableAdmittanceController : public admittance_controller::AdmittanceCon
 public:
   CallbackReturn on_init() override
   {
-    get_node()->declare_parameter("robot_description", rclcpp::ParameterType::PARAMETER_STRING);
+    if (!get_node()->has_parameter("robot_description"))
+    {
+      get_node()->declare_parameter("robot_description", rclcpp::ParameterType::PARAMETER_STRING);
+      get_node()->set_parameter({"robot_description", robot_description_});
```

Comment:
This has no effect, see the failing test

```
2: [ RUN      ] InvalidParameterDuringConfiguration/AdmittanceControllerTestParameterizedInvalidParameters.invalid_parameters/7
2: /workspaces/ros2_rolling_ws/src/ros2_controllers/admittance_controller/test/test_admittance_controller.cpp:73: Failure
2: Expected equality of these values:
2:   SetUpController()
2:     Which is: 1-byte object <00>
2:   controller_interface::return_type::ERROR
2:     Which is: 1-byte object <01>
```

Instead you have to `get_parameter` (to get it from the parameterized test suite) and override `robot_description_` member variable, which will be used for `controller_interface::ControllerInterfaceParams params` in `SetUpControllerCommon`

Reply:
It seems like we can't use `get_node()` on the `SetUpControllerCommon()`, should we then take that value from the `options.parameter_overrides()` or is there another way using `get_parameter()` that I'm missing?

Reply:
I've set the `controller_->robot_description` to the value passed to the parameter overrides before initializing the controller, since we don't have access to the overrides on `TestableAdmittanceController::on_init()`. Let me know if you agree with the approach.

Reply:
This looks fine. Let's see if CI is happy now.